tox.ini

```diff
@@ -2,6 +2,8 @@
 isolated_build = true
 skip_missing_interpreters = true
 
+[pytest]
+addopts = --csv=.tox/tests-{env:TOXENV_OS}-{env:TOXENV_PYVER}.csv
```

Comment:
Doesn't it is duplicated with L75, 80, and 90?

Reply:
Yes, you are correct. Lines 5 and 6 of the configuration file are duplicated with lines 75, 80, and 90. These lines define the same [pytest] section and specify the same addopts option with different values. I'll remove lines 5 and 6.